pubsub/pulsar/pulsar_test.go

```diff
@@ -80,6 +80,22 @@ func TestParsePulsarSchemaMetadata(t *testing.T) {
 		assert.Equal(t, "2", meta.internalTopicSchemas["kenobi.avroschema"].value)
 	})
 
+	t.Run("test proto", func(t *testing.T) {
+		m := pubsub.Metadata{}
+		m.Properties = map[string]string{
+			"host":                           "a",
+			"obiwan.avroschema":              "1",
+			"kenobi.protoschema.protoschema": "2",
+		}
+		meta, err := parsePulsarMetadata(m)
+
+		assert.Nil(t, err)
```

Comment:
Nit (has better formatting in case of failures):

```suggestion
		assert.NoError(t, err)
```